config/gradle/dependenciesCheck.gradle

```diff
@@ -49,8 +49,6 @@ afterEvaluate {
 
                     assert checksum == calculatedChecksum: "Checksum ($algorithm) does not match: $dependency.name"
                 }
-
-                logger.lifecycle("\u2705 $configuration: $dependency.name")
```

Comment:
Why do we need to remove this?

Reply:
Yes, imo it doesn't really add anything. It's only interesting to know when it fails